lib/modules/manager/asdf/readme.md

```diff
@@ -3,8 +3,45 @@ Keeps the [asdf](https://asdf-vm.com/manage/configuration.html#tool-versions)
 
 Because `asdf` supports the version management of many different tools, specific tool support needs to be added one by one.
 
-Only the following tools are currently supported
+The following tools are currently supported:
 
+- [awscli](https://github.com/MetricMike/asdf-awscli)
```

Comment:
I don't want to maintain this long list myself. Can we automate "updating the list"? The code already knows what's supported somehow.

Reply:
@HonkingGoose If it is ok for you, I would offer to add a README generator as a new separate PR after this one is merged. Is that ok for you?

Reply:
I'll let the maintainers decide what to do. 😉

Reply:
I've prepared a follow-up PR:

https://github.com/jshmrtn/renovate/compare/extend-asdf-support...jshmrtn:renovate:asdf_readme_generator

To avoid complicating matters, I will open that one as soon as this one is merged if no maintainer disagrees with this decision.

Reply:
all tooling needs to be inside `tools` folder, so it doesn't get compiled to the npm package.

Reply:
@viceice Is this something that you want to adress in this PR already or is it ok for an upcoming PR?

Is there something else that needs to be resolved before merging this PR?

Reply:
yes, seperate PR is ok